publish/working-with-pipelines/working-with-pipelines.md

```diff
@@ -309,6 +309,76 @@ You can also see pods for the pipeline runs, for which you can specify `oc descr
 
 If the pipeline run was successful, you can see a Docker image in our Docker registry and a pod that’s running your application.
 
+## Transport layer security (TLS) verification for image registry access in pipelines
+
+when accessing image registries to pull and push images, pipelines enable the configuration creation of the cluster resource, [image.config.openshift.io/cluster](https://docs.openshift.com/container-platform/4.2/openshift_images/image-configuration.html#images-configuration-file_image-configuration).  Scenarios are presented of image registries that can be used and the setup steps required.
```

Comment:
W in when should be in caps?  Also pipelines don't enable this configuration.  They use the configuration that the user did in the cluster resource.

Reply:
Updated